cmd/cortex/main.go

```diff
@@ -130,6 +130,13 @@ func main() {
 	t, err := cortex.New(cfg)
 	util.CheckFatal("initializing cortex", err)
 
+	if t.Cfg.ListModules {
+		for _, m := range t.ModuleManager.PublicModuleNames() {
+			fmt.Fprintln(flag.CommandLine.Output(), m)
```

Comment:
This should go stdout. Also, can we make the output sorted?

Reply:
Ah, sorry, didn't realize default is stderr. Will make the change. Will sort the output too.